clusterloader2/pkg/measurement/common/service_creation_latency.go

```diff
@@ -50,7 +50,10 @@ const (
 )
 
 func init() {
-	measurement.Register(serviceCreationLatencyName, createServiceCreationLatencyMeasurement)
+	if err := measurement.Register(serviceCreationLatencyName, createServiceCreationLatencyMeasurement); err != nil {
+		fmt.Errorf("cant register service %v", err)
```

Comment:
Apparently I wasn't enough explicit previously - you should use klog.Fatalf()
e.g. as here:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/profile.go#L46